tests/docker/globus.ubuntu24.Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:24.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true
+
+#### Get and install iRODS repo ####
+RUN apt-get update && apt-get install -y wget gnupg2 lsb-release && \
+    wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
+    echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list && \
+    apt-get update
```

Comment:
Do not leave cached packages or package lists behind after a `RUN` block.
Some good examples of how to properly use and clean up after `apt-get` can be found in the testing environment: https://github.com/irods/irods_testing_environment/blob/615a2e24379b1b3733d7ec4ac6ed03afe8711d47/projects/ubuntu-24.04/Dockerfile
This applies to all usages of `apt-get` in Dockerfiles.

Reply:
Done